app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -10,7 +10,7 @@ def google_oauth2
                               kind: 'Google'
       sign_in_and_redirect @user, event: :authentication
     else
-      session['devise.google_data'] = request.env['omniauth.auth']
+      session['devise.google_data'] = request.env['omniauth.auth'].except(:extra)
```

Comment:
It says to remove the extra information for the devise google_data. I don't remember why it's here (probably lifted from a more current example) but it seems fine, you can read a bit about it here: https://github.com/lynndylanhurley/devise_token_auth/issues/647